### PR TITLE
HBW-299 Use css variables for all colors

### PR DESCRIPTION
**Pull request by GideonEXE**

### CODE REVIEW

File: hbw/app/assets/stylesheets/hbw/brand_variables.sass [hunk 1]
```diff
@@ -14,3 +14,34 @@
...
+  //tasks
+  --base-hbw-inactive-task-hover-color: $hbw-inactive-task-hover-color
+  //colors
+  --base-hbw-light-gray: #ECF0F1
+  --base-hbw-red: red
+  --base-hbw-white: white
+  --base-hbw-scrollbar-color: grey white
+  --base-hbw-gray: #888
+  --base-hbw-dark-gray: #555
+  --base-hbw-blue-bayoux: #445C72
```
Comment (TimAle) [inline hbw/app/assets/stylesheets/hbw/brand_variables.sass:29]: `--base-hbw-scrollbar-color` is good naming, but `--base-hbw-blue-bayoux` is meaningless. We should use domain-related or element-related names for these variables.

File: hbw/app/assets/stylesheets/hbw/variables.sass [hunk 2]
```diff
@@ -26,3 +26,4 @@ $btn-danger-bg: transparent
 $btn-danger-color: $btn-danger-border
 
 $deadline-red: $brand-danger
+$hbw-inactive-task-hover-color: $gray-light
```
Comment (TimAle) [inline hbw/app/assets/stylesheets/hbw/variables.sass:29]: For what?

File: hbw/app/assets/stylesheets/hbw/brand_variables.sass [hunk 3]
```diff
@@ -14,3 +14,34 @@
   --base-btn-success-color: #{$btn-success-color}
   --base-btn-info-color: #{$btn-info-color}
   --base-btn-warning-color: #{$btn-warning-color}
+
+  //borders
+  --base-hbw-border-light-gray: 1px solid var(--hbw-light-gray, var(--base-hbw-light-gray))
+  //tasks
+  --base-hbw-inactive-task-hover-color: $hbw-inactive-task-hover-color
```
Comment (TimAle) [inline hbw/app/assets/stylesheets/hbw/brand_variables.sass:21]: without `#{}` this won't be sass-processed

File: CHANGELOG.md [hunk 4]
```diff
@@ -9,6 +9,9 @@ v2.4.0 [unreleased]
 - [#476](https://github.com/latera/homs/pull/476) Fix symlinking dev assets.
 - [#475](https://github.com/latera/homs/pull/475) Fix displaying errors in `select` and `select_table`.
 
+### Refactoring
+- [#481](https://github.com/latera/homs/pull/481) Use css variables for all colors
```
Comment (TimAle) [inline CHANGELOG.md:13]: The dot at the end, please

**Comment by GideonEXE** [comment]
Upd

**Pull request merged**